crates/modelardb_server/src/remote.rs

```diff
@@ -427,6 +443,7 @@ impl FlightService for FlightServiceHandler {
         &self,
         request: Request<Ticket>,
     ) -> StdResult<Response<Self::DoGetStream>, Status> {
+        let request_metadata = request.metadata().clone();
```

Comment:
Unclear why the `MetadataMap` is cloned when the type of `validate_request`'s argument is `&MetadataMap` which [`Request.metadata()`](https://docs.rs/tonic/latest/tonic/struct.Request.html#method.metadata) return.

Reply:
It was required because of the call to `request.into_inner()` on the next line but changing it to `request.get_ref()` fixed the issue.